modules/node/handler/cms_node_pagination_builder.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Paginator builder for CMS nodes."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_NODE_PAGINATION_BUILDER
```

Comment:
I am wondering, if this class does really make sense.
as well as other  CMS_NODE_API.nodes : LIST [..]
I guess we should always pass a content type
because I doubt we will really need to list any node .. but more nodes for a specific type.

Do you see my point?